### DISCUSSION TITLE
Reactivated without nix and yarn

### DISCUSSION THREAD

**Discussion by stlk** [silviogutierrez/reactivated#159] [Show and tell]
Managed to spin up your example app without nix and yarn. Wow, that's really impressive project. Especially the type safety between python and typescript. Thank you!

https://github.com/stlk/reactivated-playground

**Comment by silviogutierrez** [comment]
Thanks! I'm guessing if I update to `node_modules/.bin/tsc`, I can spare people the monkey patching.

**Reply by SerJoG** [reply]
When do you plan to do this?

**Comment by silviogutierrez** [comment]
@stlk : I've also added a Docker option you may want to try. Check out reactivated.io for more information.
